sdk/core/azure-core-tracing-opencensus/setup.py

```diff
@@ -44,7 +44,6 @@
         "Programming Language :: Python",
         "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
```

Comment:
Do you want to add "Programming Language :: Python :: 3.11",?

Reply:
Ah, good catch. Added a line for 3.11.